packages/hooks/feed/useFeedPosting.ts

```diff
@@ -74,34 +81,74 @@ export const useFeedPosting = (
       };
 
       if (isDAO) {
-        const network = mustGetCosmosNetwork(networkId);
+        if (network?.kind === NetworkKind.Gno) {
+          if (!selectedWallet) {
+            throw new Error("No wallet selected");
+          }
+
+          const msg: GnoCreatePostMessage = {
+            type: "gno.land/r/teritori/social_feeds.CreatePost",
+            payload: {
+              feedId: TERITORI_FEED_ID,
+              parentId: "0",
+              category: category.toString(),
+              metadata,
+            },
+          };
 
-        if (!network.socialFeedContractAddress) {
-          throw new Error("Social feed contract address not found");
-        }
-        if (!userAddress) {
-          throw new Error("Invalid sender");
-        }
-        setStep(feedPostingStep(FeedPostingStepId.PROPOSING));
-
-        await makeProposal(userAddress, {
-          title: "Post on feed",
-          description: "",
-          msgs: [
-            {
-              wasm: {
-                execute: {
-                  contract_addr: network.socialFeedContractAddress,
-                  msg: Buffer.from(
-                    JSON.stringify({ create_post: msg }),
-                  ).toString("base64"),
-                  funds: [{ amount: postFee.toString(), denom: "utori" }],
+          const propReq: GnoSingleChoiceProposal = {
+            title: "Post on feed",
+            description: JSON.stringify(msg),
+            messages: [msg],
+          };
+
+          setStep(feedPostingStep(FeedPostingStepId.PROPOSING));
+          const vmCall = {
+            caller: selectedWallet.address,
+            send: "",
+            pkg_path: userAddress,
+            func: "ProposeJSON",
+            args: ["0", JSON.stringify(propReq)],
+          };
+          const txHash = await adenaDoContract(
+            network.id,
+            [{ type: AdenaDoContractMessageType.CALL, value: vmCall }],
+            { gasWanted: 20_000_000 },
+          );
+          const provider = new GnoJSONRPCProvider(network.endpoint);
+          await provider.waitForTransaction(txHash);
```

Comment:
```suggestion
          await adenaDoContract(
            network.id,
            [{ type: AdenaDoContractMessageType.CALL, value: vmCall }],
            { gasWanted: 20_000_000 },
          );
```

`adenaDoContract` already waits on the tx